src/Components/Contributors/InstututionHelper.ts

```diff
@@ -86,7 +87,8 @@ export async function replaceNonCristinInstitution(affiliation: Affiliation): Pr
       }
     }
   } else if (affiliation.cristinInstitutionNr?.toString() !== '0') {
-    affiliation.institutionName = await getInstitutionName(affiliation.cristinInstitutionNr, SearchLanguage.En);
+    const clonedAffiliation = clone(affiliation);
+    clonedAffiliation.institutionName = await getInstitutionName(affiliation.cristinInstitutionNr, SearchLanguage.En);
```

Comment:
Det er antageligvis denne linja som gjør susen, men synes cloninga er grei for å holde koden til PURE kode elelrs, så har ikke fjernet alle clonene ellers.